### PR TITLE
SIP-43 fix: add missing index on "processed" columns

### PR DESCRIPTION
**Pull request by iBotPeaches**

### CODE REVIEW

File: database/migrations/2020_12_09_000004_add_missing_indexes.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class AddMissingIndexes extends Migration
+{
+    public function up(): void
+    {
+        if (DB::connection()->getDriverName() !== 'mysql') {
+            throw new \InvalidArgumentException("MySQL is the only supported driver for this package.");
+        }
```
Comment (ToothlessRebel) [inline database/migrations/2020_12_09_000004_add_missing_indexes.php:14]: I'm starting to think that every migration is not the place to check this. If the first migration threw this exception, you either fixed something or abandoned this as an option. If you changed databases between the first migration and this migration, you must have figured out how to make not MySQL work?

Comment (ToothlessRebel) [reply database/migrations/2020_12_09_000004_add_missing_indexes.php:14]: If nothing else, this is repeated several times and could be DRY'd up.

**Review by ToothlessRebel — APPROVED** [review]

File: database/migrations/2020_12_09_000004_add_missing_indexes.php (hunk 1 above)
Comment (iBotPeaches) [reply database/migrations/2020_12_09_000004_add_missing_indexes.php:14]: I think just removing it entirely and just saying we only support mysql (in readme) works for me.

Comment (ToothlessRebel) [reply database/migrations/2020_12_09_000004_add_missing_indexes.php:14]: Same end result as far as I can see, an exception gets thrown that tells you what needs to be done. You decide if how you want to handle it. It's just either a PDO exception from an unsupported database operation, or an invalid argument.

Comment (iBotPeaches) [reply database/migrations/2020_12_09_000004_add_missing_indexes.php:14]: Okay - I cleared them out and updated readme.

**Review by joeladam518 — APPROVED** [review]

**Review by ToothlessRebel — APPROVED** [review]

**Review by nicomaurer — APPROVED** [review]

**Pull request merged**